backend/Modules/Users/HitMeApp.Users/Cryptography/Crypto.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Security.Cryptography;
+
+namespace HitMeApp.Users.Cryptography
+{
+    internal static class Crypto
```

Comment:
1. Keeping it static makes no sense in the long run as it would only be needed to be instantiated when the user wants to login.
2. It makes sense to wrap whatever the whatever encryption method we use in our own adapter to provide a convenient way to replace it. It's not that trivial for the app that is developed for the long time, but in the beginning phase of the project it's extremely convient to have it hooked up in a flexible manner.
3. .NET finally provided some convenient implementation of its own called `IPasswordHasher<T>`. I think we should wrap it around our own interface. 

To sum it up: I would wrap `IPasswordHasher<T>` in our own adapter and register this adapter as `InstancePerLifetimeScope` in module's IoC container.

Reply:
Not applicable anymore. As suggested, I've used default `IPasswordHasher<T>` implementation and got rid of our custom `Crypto` class. No need to reinvent the wheel and overcomplicate things at the very beginning. Default implementation has been registered in IoC container.